### PR TITLE
[4.0] Quickicons - Add Shortcut to Template Code

### PR DESCRIPTION
**Pull request by coolcat-creations**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25916

### Summary of Changes
Added Template Code Quickicon
Changed Language Keys of Template Style Icon 
Added Language Keys for Template Code

### Testing Instructions
Apply the patch
Activate the Quick Icon in the module settings
Click the Template Code Link to see if you end up where you should

### CODE REVIEW

File: administrator/modules/mod_quickicon/mod_quickicon.xml [hunk 1]
```diff
@@ -147,15 +147,25 @@
...
 					<option value="0">JHIDE</option>
 					<option value="1">JSHOW</option>
 				</field>
+				<field
+						name="show_template_code"
+						type="radio"
+						label="MOD_QUICKICON_SHOW_TEMPLATE_CODE_LABEL"
+						layout="joomla.form.field.radio.switcher"
+						default="0"
+				>
```
Comment (Quy) [inline administrator/modules/mod_quickicon/mod_quickicon.xml]:
```suggestion
					name="show_template_code"
					type="radio"
					label="MOD_QUICKICON_SHOW_TEMPLATE_CODE_LABEL"
					layout="joomla.form.field.radio.switcher"
					default="0"
					>
```

File: administrator/templates/atum/scss/vendor/bootstrap/_badge.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
   border: 1px solid lighten($danger-txt, 30%);
 }
 
-.badge-secondary,
+.badge-secondary
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/bootstrap/_badge.scss]: Revert.

**Comment by brianteeman** [comment]
Personally I am not in favour of this.
1. You are not likely to need to do this on a regular basis. If we add this then why not add quickicons for everything (joke but I hope you see my point)
2. I still prefer the idea of a unified list for the template and the template styles

**Comment by coolcat-creations** [comment]
> Personally I am not in favour of this.
> 
>     1. You are not likely to need to do this on a regular basis. If we add this then why not add quickicons for everything (joke but I hope you see my point)

yes I agree on what you say - still this was a user request and I added it... If more people want to have this it can be easily merged. More flexible would be to just let the user add own quick icons by any way, no idea how. Maybe a subform?

>     2. I still prefer the idea of a unified list for the template and the template styles

Yes I'd like that too, but no idea how to solve it.

**Comment by brianteeman** [comment]
> > 2. I still prefer the idea of a unified list for the template and the template styles
> Yes I'd like that too, but no idea how to solve it.

Not that hard I just dont have the time at the moment

**Comment by ciar4n** [comment]
Imo one of joomlas strongest features is overidable views. It seems to be often overlooked by users. Something like this might help to resolve this.

**Comment by roland-d** [comment]
I have tested this item :white_check_mark: successfully on 4deddea7a7495f4535874bc9d590246ed5e8fabe<br /><br />I have tested this patch successfully by applying the patch. The test instructions were not so clear but what I did was, apply the patch and then go to the Modules and filtered on Quick Icons type. Then open the System module. At the bottom you find the new setting Templates Code. Enable the option and click on Save & Close. After that go to the Dashboard and in the top right corner you will see the System module with the link to the Templates code.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28977">issues.joomla.org/tracker/joomla-cms/28977</a>.</sub>

**Comment by Johanv74** [comment]
I have tested this item :white_check_mark: successfully on 4deddea7a7495f4535874bc9d590246ed5e8fabe<br /><br />Together with other JUG040 members and Roland Dalmulder we tested this patch with succes.
I tested it on my own Joomla! 4 installation.
We tested this patch successfully :-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28977">issues.joomla.org/tracker/joomla-cms/28977</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28977">issues.joomla.org/tracker/joomla-cms/28977</a>.</sub>

**Comment by zero-24** [comment]
LGTM thanks @coolcat-creations

**Pull request merged**